crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -322,26 +324,27 @@ fn spawn_app_signals_handler(
         }
     });
 
-    // TODO - metrics to indicate if we're getting overloaded here.
-    task_list
-        .lock()
-        .push(tokio::task::spawn(rx_from_cell.for_each_concurrent(
-            CONCURRENCY_COUNT,
-            move |signal| {
-                let tx_to_iface = tx_to_iface.clone();
-                async move {
-                    trace!(msg = "Sending signal!", ?signal);
-                    if let Err(err) = async move {
-                        tx_to_iface.signal(signal).await?;
-                        InterfaceResult::Ok(())
-                    }
-                    .await
-                    {
-                        error!(?err, "error emitting signal");
```

Comment:
The problem was here and how this code interacts (or doesn't) with the unfold above. As long as the cell stays enabled, the stream of signals from the cell won't end. But we do also need to stop listening for signals from the cell if the client for this connection goes away.